### ISSUE TITLE
Unable to complete DNS challenge for subdomains with Constellix

### ISSUE THREAD

**Issue by jsnjack** [go-acme/lego#1220]
DNS challenge for something like `example.com` works well with constellix. However, if I try to get the certificate for `dev.example.com` (subdomain of example.com) it won't work. I checked TXT records in constellix dashboard and for the case of `dev.example.com` they are not created

**Comment by ldez** [comment]
Hello,

could you provide:
- the command line args
- the logs
- and all information that can be useful (DNS records info, ...)

**Comment by jsnjack** [comment]
Oh yes, apologies for the lack of information in ticket description!

lego version:
lego version 3.8.0 linux/amd64

Logs and command (original domain name is replaced):
``` 
[       2ms] lego -a --domains=dev.example.com --domains=*.dev.example.com --email=admin@email.com --dns constellix --pem run
[       7ms] 2020/07/16 16:32:11 No key found for account admin@email.com. Generating a P384 key.
[      12ms] 2020/07/16 16:32:11 Saved key to /home/client/wakedir/workspace/332/.lego/accounts/acme-v02.api.letsencrypt.org/admin@email.com/keys/admin@email.com.key
[     619ms] 2020/07/16 16:32:11 [INFO] acme: Registering account for admin@email.com
[    1.035s] !!!! HEADS UP !!!!
[    1.035s] 
[    1.035s] configuration directory at "/home/client/wakedir/workspace/332/.lego/accounts".
[    1.035s] 
[    1.035s] You should make a secure backup of this folder now. This
[    1.035s] private keys obtained from Let's Encrypt so making regular
[    1.035s] backups of this folder is ideal.
[    1.035s] 2020/07/16 16:32:12 [INFO] [dev.example.com, *.dev.example.com] acme: Obtaining bundled SAN certificate
[     2.05s] 2020/07/16 16:32:13 [INFO] [*.dev.example.com] AuthURL: https://acme-v02.api.letsencrypt.org/acme/authz-v3/5918387253
[     2.05s] 2020/07/16 16:32:13 [INFO] [*.dev.example.com] acme: use dns-01 solver
[     2.05s] 2020/07/16 16:32:13 [INFO] [dev.example.com] acme: Could not find solver for: http-01
[     2.05s] 2020/07/16 16:32:13 [INFO] [dev.example.com] acme: use dns-01 solver
[     2.05s] 2020/07/16 16:32:13 [INFO] [*.dev.example.com] acme: Preparing to solve DNS-01
[    4.171s] 2020/07/16 16:32:15 [INFO] [dev.example.com] acme: Preparing to solve DNS-01
[    5.084s] 2020/07/16 16:32:16 [INFO] [*.dev.example.com] acme: Trying to solve DNS-01
[    5.084s] 2020/07/16 16:32:16 [INFO] [*.dev.example.com] acme: Checking DNS record propagation using [1.1.1.1:53 8.8.8.8:53 [2606:4700:4700::1111]:53 [2606:4700:4700::1001]:53]
[   15.085s] 2020/07/16 16:32:26 [INFO] Wait for propagation [timeout: 10m0s, interval: 10s]
[   15.132s] 2020/07/16 16:32:26 [INFO] [*.dev.example.com] acme: Waiting for DNS record propagation.
[   25.176s] 2020/07/16 16:32:36 [INFO] [*.dev.example.com] acme: Waiting for DNS record propagation.
[   35.223s] 2020/07/16 16:32:46 [INFO] [*.dev.example.com] acme: Waiting for DNS record propagation.
[   45.266s] 2020/07/16 16:32:56 [INFO] [*.dev.example.com] acme: Waiting for DNS record propagation.
[   55.311s] 2020/07/16 16:33:06 [INFO] [*.dev.example.com] acme: Waiting for DNS record propagation.
[  1m5.352s] 2020/07/16 16:33:16 [INFO] [*.dev.example.com] acme: Waiting for DNS record propagation.
...
```

When I try to get the certificate for dev.example.com, TXT record is not created (for example.com it works fine)

**Comment by ldez** [comment]
Could you provide:
- the full logs
- DNS records

FYI it works for me.

**Comment by jsnjack** [comment]
> FYI it works for me.
interesting, I've tried it on 2 different domain names and it didn't work..

Full logs:
```
[       2ms] lego -a --domains=dev.example.com --domains=*.dev.example.com --email=admin@mail.com --dns constellix --pem run
[       7ms] 2020/07/16 16:32:11 No key found for account admin@mail.com. Generating a P384 key.
[      12ms] 2020/07/16 16:32:11 Saved key to /home/client/wakedir/workspace/332/.lego/accounts/acme-v02.api.letsencrypt.org/admin@mail.com/keys/admin@mail.com.key
[     619ms] 2020/07/16 16:32:11 [INFO] acme: Registering account for admin@mail.com
[    1.035s] !!!! HEADS UP !!!!
[    1.035s] 
[    1.035s] Your account credentials have been saved in your Let's Encrypt
[    1.035s] configuration directory at "/home/client/wakedir/workspace/332/.lego/accounts".
[    1.035s] 
[    1.035s] You should make a secure backup of this folder now. This
[    1.035s] configuration directory will also contain certificates and
[    1.035s] private keys obtained from Let's Encrypt so making regular
[    1.035s] backups of this folder is ideal.
[    1.035s] 2020/07/16 16:32:12 [INFO] [dev.example.com, *.dev.example.com] acme: Obtaining bundled SAN certificate
[     2.05s] 2020/07/16 16:32:13 [INFO] [*.dev.example.com] AuthURL: https://acme-v02.api.letsencrypt.org/acme/authz-v3/5918387253
[     2.05s] 2020/07/16 16:32:13 [INFO] [dev.example.com] AuthURL: https://acme-v02.api.letsencrypt.org/acme/authz-v3/5918387271
[     2.05s] 2020/07/16 16:32:13 [INFO] [*.dev.example.com] acme: use dns-01 solver
[     2.05s] 2020/07/16 16:32:13 [INFO] [dev.example.com] acme: Could not find solver for: tls-alpn-01
[     2.05s] 2020/07/16 16:32:13 [INFO] [dev.example.com] acme: Could not find solver for: http-01
[     2.05s] 2020/07/16 16:32:13 [INFO] [dev.example.com] acme: use dns-01 solver
[     2.05s] 2020/07/16 16:32:13 [INFO] [*.dev.example.com] acme: Preparing to solve DNS-01
[    4.171s] 2020/07/16 16:32:15 [INFO] [dev.example.com] acme: Preparing to solve DNS-01
[    5.084s] 2020/07/16 16:32:16 [INFO] [*.dev.example.com] acme: Trying to solve DNS-01
[    5.084s] 2020/07/16 16:32:16 [INFO] [*.dev.example.com] acme: Checking DNS record propagation using [1.1.1.1:53 8.8.8.8:53 [2606:4700:4700::1111]:53 [2606:4700:4700::1001]:53]
[   15.085s] 2020/07/16 16:32:26 [INFO] Wait for propagation [timeout: 10m0s, interval: 10s]
[   15.132s] 2020/07/16 16:32:26 [INFO] [*.dev.example.com] acme: Waiting for DNS record propagation.
[   25.176s] 2020/07/16 16:32:36 [INFO] [*.dev.example.com] acme: Waiting for DNS record propagation.
[   35.223s] 2020/07/16 16:32:46 [INFO] [*.dev.example.com] acme: Waiting for DNS record propagation.
[   45.266s] 2020/07/16 16:32:56 [INFO] [*.dev.example.com] acme: Waiting for DNS record propagation.
[   55.311s] 2020/07/16 16:33:06 [INFO] [*.dev.example.com] acme: Waiting for DNS record propagation.
[  1m5.352s] 2020/07/16 16:33:16 [INFO] [*.dev.example.com] acme: Waiting for DNS record propagation.
[  1m15.41s] 2020/07/16 16:33:26 [INFO] [*.dev.example.com] acme: Waiting for DNS record propagation.
[ 1m25.452s] 2020/07/16 16:33:36 [INFO] [*.dev.example.com] acme: Waiting for DNS record propagation.
[ 1m35.494s] 2020/07/16 16:33:46 [INFO] [*.dev.example.com] acme: Waiting for DNS record propagation.
[ 1m45.552s] 2020/07/16 16:33:56 [INFO] [*.dev.example.com] acme: Waiting for DNS record propagation.
[ 1m55.598s] 2020/07/16 16:34:06 [INFO] [*.dev.example.com] acme: Waiting for DNS record propagation.
[  2m5.639s] 2020/07/16 16:34:17 [INFO] [*.dev.example.com] acme: Waiting for DNS record propagation.
[ 2m15.681s] 2020/07/16 16:34:27 [INFO] [*.dev.example.com] acme: Waiting for DNS record propagation.
[ 2m25.726s] 2020/07/16 16:34:37 [INFO] [*.dev.example.com] acme: Waiting for DNS record propagation.
[ 2m35.767s] 2020/07/16 16:34:47 [INFO] [*.dev.example.com] acme: Waiting for DNS record propagation.
[  2m45.81s] 2020/07/16 16:34:57 [INFO] [*.dev.example.com] acme: Waiting for DNS record propagation.
[ 2m55.853s] 2020/07/16 16:35:07 [INFO] [*.dev.example.com] acme: Waiting for DNS record propagation.
[  3m5.896s] 2020/07/16 16:35:17 [INFO] [*.dev.example.com] acme: Waiting for DNS record propagation.
[ 3m15.945s] 2020/07/16 16:35:27 [INFO] [*.dev.example.com] acme: Waiting for DNS record propagation.
[ 3m25.988s] 2020/07/16 16:35:37 [INFO] [*.dev.example.com] acme: Waiting for DNS record propagation.
[ 3m36.038s] 2020/07/16 16:35:47 [INFO] [*.dev.example.com] acme: Waiting for DNS record propagation.
[ 3m46.082s] 2020/07/16 16:35:57 [INFO] [*.dev.example.com] acme: Waiting for DNS record propagation.
[ 3m56.124s] 2020/07/16 16:36:07 [INFO] [*.dev.example.com] acme: Waiting for DNS record propagation.
[  4m6.167s] 2020/07/16 16:36:17 [INFO] [*.dev.example.com] acme: Waiting for DNS record propagation.
[ 4m16.211s] 2020/07/16 16:36:27 [INFO] [*.dev.example.com] acme: Waiting for DNS record propagation.
[ 4m26.257s] 2020/07/16 16:36:37 [INFO] [*.dev.example.com] acme: Waiting for DNS record propagation.
[ 4m36.299s] 2020/07/16 16:36:47 [INFO] [*.dev.example.com] acme: Waiting for DNS record propagation.
[ 4m46.342s] 2020/07/16 16:36:57 [INFO] [*.dev.example.com] acme: Waiting for DNS record propagation.
[ 4m56.385s] 2020/07/16 16:37:07 [INFO] [*.dev.example.com] acme: Waiting for DNS record propagation.
[  5m6.432s] 2020/07/16 16:37:17 [INFO] [*.dev.example.com] acme: Waiting for DNS record propagation.
[ 5m16.478s] 2020/07/16 16:37:27 [INFO] [*.dev.example.com] acme: Waiting for DNS record propagation.
[ 5m26.525s] 2020/07/16 16:37:37 [INFO] [*.dev.example.com] acme: Waiting for DNS record propagation.
[ 5m36.568s] 2020/07/16 16:37:47 [INFO] [*.dev.example.com] acme: Waiting for DNS record propagation.
[ 5m46.614s] 2020/07/16 16:37:57 [INFO] [*.dev.example.com] acme: Waiting for DNS record propagation.
[ 5m56.657s] 2020/07/16 16:38:08 [INFO] [*.dev.example.com] acme: Waiting for DNS record propagation.
[  6m6.701s] 2020/07/16 16:38:18 [INFO] [*.dev.example.com] acme: Waiting for DNS record propagation.
[ 6m16.749s] 2020/07/16 16:38:28 [INFO] [*.dev.example.com] acme: Waiting for DNS record propagation.
[ 6m26.791s] 2020/07/16 16:38:38 [INFO] [*.dev.example.com] acme: Waiting for DNS record propagation.
[ 6m36.834s] 2020/07/16 16:38:48 [INFO] [*.dev.example.com] acme: Waiting for DNS record propagation.
[ 6m46.881s] 2020/07/16 16:38:58 [INFO] [*.dev.example.com] acme: Waiting for DNS record propagation.
[ 6m56.923s] 2020/07/16 16:39:08 [INFO] [*.dev.example.com] acme: Waiting for DNS record propagation.
[  7m6.966s] 2020/07/16 16:39:18 [INFO] [*.dev.example.com] acme: Waiting for DNS record propagation.
[ 7m17.008s] 2020/07/16 16:39:28 [INFO] [*.dev.example.com] acme: Waiting for DNS record propagation.
[ 7m27.049s] 2020/07/16 16:39:38 [INFO] [*.dev.example.com] acme: Waiting for DNS record propagation.
[ 7m37.093s] 2020/07/16 16:39:48 [INFO] [*.dev.example.com] acme: Waiting for DNS record propagation.
[ 7m47.135s] 2020/07/16 16:39:58 [INFO] [*.dev.example.com] acme: Waiting for DNS record propagation.
[ 7m57.179s] 2020/07/16 16:40:08 [INFO] [*.dev.example.com] acme: Waiting for DNS record propagation.
[  8m7.222s] 2020/07/16 16:40:18 [INFO] [*.dev.example.com] acme: Waiting for DNS record propagation.
[ 8m17.267s] 2020/07/16 16:40:28 [INFO] [*.dev.example.com] acme: Waiting for DNS record propagation.
[  8m27.31s] 2020/07/16 16:40:38 [INFO] [*.dev.example.com] acme: Waiting for DNS record propagation.
[ 8m37.353s] 2020/07/16 16:40:48 [INFO] [*.dev.example.com] acme: Waiting for DNS record propagation.
[ 8m47.395s] 2020/07/16 16:40:58 [INFO] [*.dev.example.com] acme: Waiting for DNS record propagation.
[ 8m57.437s] 2020/07/16 16:41:08 [INFO] [*.dev.example.com] acme: Waiting for DNS record propagation.
[  9m7.478s] 2020/07/16 16:41:18 [INFO] [*.dev.example.com] acme: Waiting for DNS record propagation.
[  9m17.53s] 2020/07/16 16:41:28 [INFO] [*.dev.example.com] acme: Waiting for DNS record propagation.
[ 9m27.572s] 2020/07/16 16:41:38 [INFO] [*.dev.example.com] acme: Waiting for DNS record propagation.
[ 9m37.614s] 2020/07/16 16:41:48 [INFO] [*.dev.example.com] acme: Waiting for DNS record propagation.
[  9m47.66s] 2020/07/16 16:41:59 [INFO] [*.dev.example.com] acme: Waiting for DNS record propagation.
[ 9m57.702s] 2020/07/16 16:42:09 [INFO] [*.dev.example.com] acme: Waiting for DNS record propagation.
[ 10m7.745s] 2020/07/16 16:42:19 [INFO] [*.dev.example.com] acme: Waiting for DNS record propagation.
[10m17.746s] 2020/07/16 16:42:29 [INFO] [dev.example.com] acme: Trying to solve DNS-01
[10m17.746s] 2020/07/16 16:42:29 [INFO] [dev.example.com] acme: Checking DNS record propagation using [1.1.1.1:53 8.8.8.8:53 [2606:4700:4700::1111]:53 [2606:4700:4700::1001]:53]
[10m27.746s] 2020/07/16 16:42:39 [INFO] Wait for propagation [timeout: 10m0s, interval: 10s]
[10m27.788s] 2020/07/16 16:42:39 [INFO] [dev.example.com] acme: Waiting for DNS record propagation.
[10m37.834s] 2020/07/16 16:42:49 [INFO] [dev.example.com] acme: Waiting for DNS record propagation.
[10m47.876s] 2020/07/16 16:42:59 [INFO] [dev.example.com] acme: Waiting for DNS record propagation.
[10m57.919s] 2020/07/16 16:43:09 [INFO] [dev.example.com] acme: Waiting for DNS record propagation.
[ 11m7.963s] 2020/07/16 16:43:19 [INFO] [dev.example.com] acme: Waiting for DNS record propagation.
[11m18.004s] 2020/07/16 16:43:29 [INFO] [dev.example.com] acme: Waiting for DNS record propagation.
[11m28.045s] 2020/07/16 16:43:39 [INFO] [dev.example.com] acme: Waiting for DNS record propagation.
[11m38.089s] 2020/07/16 16:43:49 [INFO] [dev.example.com] acme: Waiting for DNS record propagation.
[11m48.131s] 2020/07/16 16:43:59 [INFO] [dev.example.com] acme: Waiting for DNS record propagation.
[11m58.174s] 2020/07/16 16:44:09 [INFO] [dev.example.com] acme: Waiting for DNS record propagation.
[ 12m8.216s] 2020/07/16 16:44:19 [INFO] [dev.example.com] acme: Waiting for DNS record propagation.
[12m18.263s] 2020/07/16 16:44:29 [INFO] [dev.example.com] acme: Waiting for DNS record propagation.
[12m28.306s] 2020/07/16 16:44:39 [INFO] [dev.example.com] acme: Waiting for DNS record propagation.
[ 12m38.35s] 2020/07/16 16:44:49 [INFO] [dev.example.com] acme: Waiting for DNS record propagation.
[12m48.393s] 2020/07/16 16:44:59 [INFO] [dev.example.com] acme: Waiting for DNS record propagation.
[ 12m58.44s] 2020/07/16 16:45:09 [INFO] [dev.example.com] acme: Waiting for DNS record propagation.
[ 13m8.482s] 2020/07/16 16:45:19 [INFO] [dev.example.com] acme: Waiting for DNS record propagation.
[13m18.525s] 2020/07/16 16:45:29 [INFO] [dev.example.com] acme: Waiting for DNS record propagation.
[13m28.569s] 2020/07/16 16:45:39 [INFO] [dev.example.com] acme: Waiting for DNS record propagation.
[13m38.611s] 2020/07/16 16:45:49 [INFO] [dev.example.com] acme: Waiting for DNS record propagation.
[13m48.653s] 2020/07/16 16:46:00 [INFO] [dev.example.com] acme: Waiting for DNS record propagation.
[13m58.696s] 2020/07/16 16:46:10 [INFO] [dev.example.com] acme: Waiting for DNS record propagation.
[ 14m8.739s] 2020/07/16 16:46:20 [INFO] [dev.example.com] acme: Waiting for DNS record propagation.
[14m18.782s] 2020/07/16 16:46:30 [INFO] [dev.example.com] acme: Waiting for DNS record propagation.
[14m28.825s] 2020/07/16 16:46:40 [INFO] [dev.example.com] acme: Waiting for DNS record propagation.
[14m38.867s] 2020/07/16 16:46:50 [INFO] [dev.example.com] acme: Waiting for DNS record propagation.
[14m48.911s] 2020/07/16 16:47:00 [INFO] [dev.example.com] acme: Waiting for DNS record propagation.
[14m58.954s] 2020/07/16 16:47:10 [INFO] [dev.example.com] acme: Waiting for DNS record propagation.
[ 15m8.998s] 2020/07/16 16:47:20 [INFO] [dev.example.com] acme: Waiting for DNS record propagation.
[15m19.056s] 2020/07/16 16:47:30 [INFO] [dev.example.com] acme: Waiting for DNS record propagation.
[15m29.098s] 2020/07/16 16:47:40 [INFO] [dev.example.com] acme: Waiting for DNS record propagation.
[15m39.141s] 2020/07/16 16:47:50 [INFO] [dev.example.com] acme: Waiting for DNS record propagation.
[15m49.184s] 2020/07/16 16:48:00 [INFO] [dev.example.com] acme: Waiting for DNS record propagation.
[15m59.228s] 2020/07/16 16:48:10 [INFO] [dev.example.com] acme: Waiting for DNS record propagation.
[ 16m9.273s] 2020/07/16 16:48:20 [INFO] [dev.example.com] acme: Waiting for DNS record propagation.
[16m19.317s] 2020/07/16 16:48:30 [INFO] [dev.example.com] acme: Waiting for DNS record propagation.
[16m29.359s] 2020/07/16 16:48:40 [INFO] [dev.example.com] acme: Waiting for DNS record propagation.
[16m39.402s] 2020/07/16 16:48:50 [INFO] [dev.example.com] acme: Waiting for DNS record propagation.
[16m49.443s] 2020/07/16 16:49:00 [INFO] [dev.example.com] acme: Waiting for DNS record propagation.
[16m59.485s] 2020/07/16 16:49:10 [INFO] [dev.example.com] acme: Waiting for DNS record propagation.
[ 17m9.527s] 2020/07/16 16:49:20 [INFO] [dev.example.com] acme: Waiting for DNS record propagation.
[17m19.568s] 2020/07/16 16:49:30 [INFO] [dev.example.com] acme: Waiting for DNS record propagation.
[17m29.609s] 2020/07/16 16:49:40 [INFO] [dev.example.com] acme: Waiting for DNS record propagation.
[17m39.653s] 2020/07/16 16:49:51 [INFO] [dev.example.com] acme: Waiting for DNS record propagation.
[17m49.695s] 2020/07/16 16:50:01 [INFO] [dev.example.com] acme: Waiting for DNS record propagation.
[17m59.738s] 2020/07/16 16:50:11 [INFO] [dev.example.com] acme: Waiting for DNS record propagation.
[ 18m9.781s] 2020/07/16 16:50:21 [INFO] [dev.example.com] acme: Waiting for DNS record propagation.
[18m19.827s] 2020/07/16 16:50:31 [INFO] [dev.example.com] acme: Waiting for DNS record propagation.
[18m29.869s] 2020/07/16 16:50:41 [INFO] [dev.example.com] acme: Waiting for DNS record propagation.
[18m39.913s] 2020/07/16 16:50:51 [INFO] [dev.example.com] acme: Waiting for DNS record propagation.
[18m49.955s] 2020/07/16 16:51:01 [INFO] [dev.example.com] acme: Waiting for DNS record propagation.
[18m59.999s] 2020/07/16 16:51:11 [INFO] [dev.example.com] acme: Waiting for DNS record propagation.
[19m10.041s] 2020/07/16 16:51:21 [INFO] [dev.example.com] acme: Waiting for DNS record propagation.
[19m20.083s] 2020/07/16 16:51:31 [INFO] [dev.example.com] acme: Waiting for DNS record propagation.
[19m30.126s] 2020/07/16 16:51:41 [INFO] [dev.example.com] acme: Waiting for DNS record propagation.
[19m40.169s] 2020/07/16 16:51:51 [INFO] [dev.example.com] acme: Waiting for DNS record propagation.
[19m50.215s] 2020/07/16 16:52:01 [INFO] [dev.example.com] acme: Waiting for DNS record propagation.
[ 20m0.257s] 2020/07/16 16:52:11 [INFO] [dev.example.com] acme: Waiting for DNS record propagation.
[  20m10.3s] 2020/07/16 16:52:21 [INFO] [dev.example.com] acme: Waiting for DNS record propagation.
[20m20.347s] 2020/07/16 16:52:31 [INFO] [dev.example.com] acme: Waiting for DNS record propagation.
[20m30.347s] 2020/07/16 16:52:41 [INFO] [*.dev.example.com] acme: Cleaning DNS-01 challenge
[20m31.567s] 2020/07/16 16:52:42 [INFO] [dev.example.com] acme: Cleaning DNS-01 challenge
[20m34.041s] 2020/07/16 16:52:45 [INFO] retry due to: acme: error: 400 :: POST :: https://acme-v02.api.letsencrypt.org/acme/authz-v3/5918387253 :: urn:ietf:params:acme:error:badNonce :: JWS has an invalid anti-replay nonce: "0001n23x46UfPUauN88ZHapErhIbi4k8lMlbZICtjMaey5I", url: 
[20m34.337s] 2020/07/16 16:52:45 [INFO] Deactivating auth: https://acme-v02.api.letsencrypt.org/acme/authz-v3/5918387253
[20m34.754s] 2020/07/16 16:52:46 [INFO] Deactivating auth: https://acme-v02.api.letsencrypt.org/acme/authz-v3/5918387271
[20m34.949s] 2020/07/16 16:52:46 Could not obtain certificates:
[20m34.949s] 	error: one or more domains had a problem:
[ 20m34.95s] [*.dev.example.com] time limit exceeded: last error: NS ns61.constellix.net. did not return the expected TXT record [fqdn: _acme-challenge.dev.example.com., value: BvJ2MsMcTVTtkVQ03cETQJ5XdWTfQYqIDpN1GqRYNGA]: 
[ 20m34.95s] [dev.example.com] time limit exceeded: last error: NS ns61.constellix.net. did not return the expected TXT record [fqdn: _acme-challenge.dev.example.com., value: ownnxma6A0nOBtnIG_SMxJtck8G8u7HuzyAx2NYvh4g]: 
[20m34.951s] Exit code: 1
```

DNS records: there are no TXT records on the dev.example.com name. There are couple of A records, but they don't seem to be relevant (I can provide them if they are)

**Comment by jsnjack** [comment]
Found the problem (silly mistake). The issue was on my side. We use 2 DNS providers - constellix and transip. The logs I provided were slightly modified and the script that we use suppose to automatically detect the DNS provider. In this case it was detecting transip. So the real command was:
```
lego -a --domains=dev.example.com --domains=*.dev.example.com --email=admin@mail.com --dns transip --pem run
```
In this case, obviously, no records in constellix were created... :) 

Thanks for your help!
